meta-lmp-base/recipes-samples/images/lmp-feature-wayland.inc

```diff
@@ -6,3 +6,5 @@ CORE_IMAGE_BASE_INSTALL += " \
     wayland \
     ${@bb.utils.contains("DISTRO_FEATURES", "x11 wayland", "weston-xwayland", "", d)} \
 "
+
+IMAGE_FEATURES += '${@bb.utils.contains('DISTRO_FEATURES', 'wayland', ' weston', '', d)}'
```

Comment:
Might not need the extra space before weston here (' weston'). Mind checking with bitbake -e | grep IMAGE_FEATURES?

Reply:
```
bitbake -e lmp-factory-image | grep IMAGE_FEATURES=
IMAGE_FEATURES="ssh-server-openssh weston"
bitbake -e lmp-factory-image | grep SYSTEMD_DEFAULT_TARGET=
SYSTEMD_DEFAULT_TARGET="graphical.target"
```